test/testBreakCondition.cpp

```diff
@@ -261,7 +262,7 @@ TEST(ObserverFeature, TimeEvolution) {
 
   // detection two
   c.setCurrentStep(0.1);
-  c.setTrajectoryLength(10.05);
+  c.setTrajectoryLength(15.05);
```

Comment:
Here I changed the trajectory length of the candidate to a length larger then the last
detection threshold so a detection is expected.
10.05 is not enough, since 10 is not a detected length (see comment on line 239)